videostore/videostore.go

```diff
@@ -0,0 +1,216 @@
+// Package videostore stores video and allows it to be queried
+package videostore
+
+import (
+	"context"
+	"encoding/base64"
+	"errors"
+	"time"
+
+	"github.com/viam-modules/video-store/videostore"
+	"go.viam.com/rdk/components/camera"
+	"go.viam.com/rdk/components/generic"
+	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/resource"
+)
+
+const (
+	defaultSegmentSeconds      = 30 // seconds
+	defaultUploadPath          = ".viam/capture/video-upload"
+	defaultStoragePath         = ".viam/video-storage-viamrtsp"
```

Comment:
Should we delete the `defaultStoragePath` and `maxGRPCSize` consts in rtsp.go file now?

Reply:
good call, done